charts/minecraft/templates/deployment.yaml

```diff
@@ -218,7 +218,7 @@ spec:
         {{- if .service.enabled }}
         - name: {{ .name }}
           containerPort: {{ .containerPort }}
-          protocol: TCP
+          protocol: {{ .protocol }}
```

Comment:
Seems like it should default to TCP or better yet wrap `protocol` line in a conditional since the kubernetes API default is TCP when not specified.